server/controllers/finance/reports/invoicedReceivedStock/report.handlebars

```diff
@@ -0,0 +1,176 @@
+            <td class="text-right text-uppercase"><strong> {{translate 'TABLE.COLUMNS.DIFFERENCE'}} </strong></td>
+            <td class="text-right text-uppercase"> <strong>{{ debcred difference ../metadata.enterprise.currency_id }}</strong> </td>
+          </tr>
+          <tr>
+            <td colspan="9"></td>
+          </tr>
+          <tr>
+            <td colspan="9"></td>
+          </tr>
+          {{else}}
+            {{>emptyTable columns=9}}
+          {{/each}}
+        </tbody>
+      </table>
+      <br>
+      <br>
+      <br>
+      {{#if checkNoInvoiceAttribution }}
+      <table style="page-break-after: always; width: 100%; margin-right: auto; margin-left: auto; font-size:80%;" class="table table-striped table-condensed table-report table-bordered">
+        <thead>
+          <tr style="background-color:#cfcfcf;">
+            <th class="text-uppercase text-bold" colspan="5"> {{translate 'REPORT.COMPARE_INVOICED_RECEIVED.NO_INVOICE_ATTRIBUTION'}}</th>
+          </tr>
+          <tr style="background-color:#efefef; font-size:84%;">
+            <th style="width: 28%;" class="text-uppercase text-center text-bold">
+              {{translate 'TABLE.COLUMNS.ITEM'}}
+            </th>
+            <th style="width: 18%;" class="text-uppercase text-center">
+              {{translate 'TABLE.COLUMNS.DATE'}}
+            </th>
+            <th style="width: 18%;" class="text-uppercase text-center">
+              {{translate 'TABLE.COLUMNS.QUANTITY'}}
+            </th>
+            <th style="width: 18%;" class="text-uppercase text-center">
+              {{translate 'TABLE.COLUMNS.VALUE'}}
+            </th>
+            <th style="width: 18%;" class="text-uppercase text-center">
+              {{translate 'TABLE.COLUMNS.TOTAL'}}
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each noInvoiceAttributionAggregat}}
+          <tr>
+            <td class="text-uppercase text-bold" style="background-color:#cfcfcf;"  colspan="5">
+              {{ document }}
+            </td>
+          </tr>
+          {{#each inventories}}
+            <tr>
+              <td> {{ inventory_text }} </td>
+              <td> {{ timestamp date }} </td>
+              <td class="text-right"> {{ quantity }} </td>
+              <td class="text-right"> {{ debcred unit_cost ../../metadata.enterprise.currency_id }} </td>
+              <td class="text-right"> <strong>{{ debcred total_cost ../../metadata.enterprise.currency_id }}</strong> </td>
+            </tr>
+          {{/each}}
+          <tr style="background-color:#efefef;">
+            <td colspan="4" class="text-right"><strong>{{translate 'TABLE.COLUMNS.TOTAL'}}</strong></td>
+            <td class="text-right"><strong> {{ debcred total_movement ../../metadata.enterprise.currency_id }}  </strong></td>
+          </tr>
+          <tr>
+            <td colspan="5"></td>
```

Comment:
Why do you have these extra `<td>`s?  They add a kind of weird three lines below the report.  Could you just put a `padding` or `margin` on something to add space if you need it?  Or even a `<br>`?

![image](https://user-images.githubusercontent.com/896472/81285907-21fd6d80-9058-11ea-8fad-be5f44c9aa11.png)